### PR TITLE
LPS-100997 Make Asset change tracked

### PR DESCRIPTION
**Pull request by stsquared99**

### CODE REVIEW

**Comment by stsquared99** [comment]
Passed CI tests here: https://github.com/stsquared99/liferay-portal/pull/85

File: portal-kernel/src/com/liferay/portal/kernel/upgrade/UpgradeCTModel.java [hunk 1]
```diff
@@ -20,30 +20,42 @@
...
 
 /**
  * @author Preston Crary
  */
 public class UpgradeCTModel extends UpgradeProcess {
 
-	public UpgradeCTModel(String tableName) {
-		_tableName = tableName;
+	public UpgradeCTModel(String... tableNames) {
+		_tableNames = tableNames;
```
Comment (Preston-Crary) [inline portal-kernel/src/com/liferay/portal/kernel/upgrade/UpgradeCTModel.java]: Throw an error if tableNames is empty.

**Comment by stsquared99** [comment]
@Preston-Crary updated

**Comment by Preston-Crary** [comment]
Please see https://github.com/achaparro/liferay-portal/pull/376

**Pull request closed without merging**